apps/root/src/pages/dca/create-position/components/swap/index.tsx

```diff
@@ -923,102 +931,136 @@ const Swap = ({ currentNetwork, yieldOptions, isLoadingYieldOptions, handleChang
-            onAction: handleNewPosition,
-            label: intl.formatMessage({
-              description: 'transactionDCAConfirmationNewPosition',
-              defaultMessage: 'Create new position',
-            }),
-          },
-        ]}
-        txIdentifierForSatisfaction={TransactionApplicationIdentifier.DCA}
-      />
-      <TokenPicker
-        shouldShow={shouldShowPicker}
-        onClose={() => setShouldShowPicker(false)}
-        modalTitle={tokenPickerModalTitle}
-        onChange={
-          (from && selecting.address === from.address) || selecting.address === ('from' as Address)
-            ? onSetFrom
-            : onSetTo
-        }
-        yieldOptions={yieldOptions}
-        otherSelected={
-          (from && selecting.address === from.address) || selecting.address === ('from' as Address) ? to : from
-        }
-      />
-      {showFirstStep && (
-        <SwapFirstStep
-          startSelectingCoin={startSelectingCoin}
-          handleFrequencyChange={handleFrequencyChange}
-          onChangeNetwork={handleChangeNetwork}
-          handleFromValueChange={handleFromValueChange}
-          rateUsdPrice={rateUsdPrice}
-          yieldEnabled={shouldEnableYield}
-          fromCanHaveYield={fromCanHaveYield}
-          toCanHaveYield={toCanHaveYield}
           yieldOptions={yieldOptions}
-          isLoadingYieldOptions={isLoadingYieldOptions}
-          usdPrice={usdPrice}
-          onButtonClick={onButtonClick}
-          isApproved={isApproved}
-          isLoadingUsdPrice={isLoadingUsdPrice}
-          allowanceErrors={allowanceErrors}
-          existingPair={existingPair}
-          currentNetwork={currentNetwork}
+          otherSelected={
+            (from && selecting.address === from.address) || selecting.address === ('from' as Address) ? to : from
+          }
         />
+        {showFirstStep && (
+          <SwapFirstStep
+            startSelectingCoin={startSelectingCoin}
+            handleFrequencyChange={handleFrequencyChange}
+            onChangeNetwork={handleChangeNetwork}
+            handleFromValueChange={handleFromValueChange}
+            rateUsdPrice={rateUsdPrice}
+            yieldEnabled={shouldEnableYield}
+            fromCanHaveYield={fromCanHaveYield}
+            toCanHaveYield={toCanHaveYield}
+            yieldOptions={yieldOptions}
+            isLoadingYieldOptions={isLoadingYieldOptions}
+            usdPrice={usdPrice}
+            onButtonClick={onButtonClick}
+            isApproved={isApproved}
+            isLoadingUsdPrice={isLoadingUsdPrice}
+            allowanceErrors={allowanceErrors}
+            existingPair={existingPair}
+            currentNetwork={currentNetwork}
+          />
+        )}
+      </StyledPaper>
+      {shouldShowConfirmation && !!currentFullTransaction?.receipt && (
+        <StyledFrequentRecipient>
+          <ContainerBox gap={1} alignItems="center" color={colors[themeMode].typography.typo2}>
+            <ProfileAddIcon />
```

Comment:
We have some descriptions and containers named according to frequent recipient logic. Also, I agree with our designer that another icon might be more suitable